classes/person.rb

```diff
@@ -29,6 +29,11 @@ def add_rental(rental)
     @rentals.push(rental) unless @rentals.include?(rental)
   end
 
+  def create_object
+    object_data = { age: @age, name: @name, id: @id }
+    @object_data = object_data
+  end
+
```

Comment:
- [ ] the ID here is causing an issue when exiting and opening the app, it re-attributes a new ID every time

![Capture d’écran 2022-05-18 100827](https://user-images.githubusercontent.com/61562626/169003922-d2eabfe7-0b7e-42aa-b212-ed61391c0881.png)
![Capture d’écran 2022-05-18 100854](https://user-images.githubusercontent.com/61562626/169003928-8b591c58-f13d-46c3-900a-e649acd70b9e.png)

Reply:
Hi @dasileker 

![Screenshot (253)](https://user-images.githubusercontent.com/88584398/169026796-ed841dbc-d180-4fb4-9484-0f0aaa4794e6.png)
Kindly suggest a new way I can make the `ID` to be unique rather than it being random?

Reply:
hello @Tobinchilongo the ID in the logic file is generated every time when you exit and open, I pointed to the one that is relative to one of the app.rb